docs/developers/building.rst

```diff
@@ -9,23 +9,36 @@ want to test your changes locally.
 Build requirements
 ------------------
 
-MicroOVN is distributed as a snap and the only requirements for building it
-are ``Make`` and ``snapcraft``. You can install them with:
+MicroOVN is distributed as a snap and most of the build requirements are handled
+by ``snapcraft`` automatically. However your build host will need:
 
-.. code-block:: none
+* make
+* autoconf
+* snapcraft
+* lxd
 
-   sudo apt install make
-   sudo snap install snapcraft --classic
```

Comment:
What is the reasoning behind removing all of the examples for how to install the various required components?

Reply:
It makes the documentation more universally applicable by not assuming user's package manager.